src/client/app/warehouses/warehouses.directives.js

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
src/client/app/warehouses/warehouses.directives.js
  line 13  col 11  Missing semicolon.             (W033)
  line 19  col 8   Missing semicolon.             (W033)
  line 35  col 15  Strings must use singlequote.  (W109)
  line 36  col 20  Strings must use singlequote.  (W109)
  line 37  col 21  Strings must use singlequote.  (W109)
  line 61  col 19  Strings must use singlequote.  (W109)
  line 62  col 20  Strings must use singlequote.  (W109)
  line 62  col 41  Strings must use singlequote.  (W109)
  line 63  col 24  Strings must use singlequote.  (W109)
  line 63  col 38  Strings must use singlequote.  (W109)
  line 63  col 92  Strings must use singlequote.  (W109)
  line 66  col 19  Strings must use singlequote.  (W109)
  line 67  col 20  Strings must use singlequote.  (W109)
  line 67  col 39  Strings must use singlequote.  (W109)
  line 40  col 17  'iopctrl' is not defined.      (W117)
  line 43  col 18  'iopctrl' is not defined.      (W117)
  line 55  col 22  'iopctrl' is not defined.      (W117)

  ⚠  17 warnings